Pidaq/Lib/wind_tunnel/air_properties.py

```diff
@@ -0,0 +1,112 @@
+"""Contains class that defines linear interpolation functions for wind tunnel testing
+
+Classes:
+    LinearInterpolation
+"""
+
+import csv
+import logging
+import sys
+from scipy import interpolate
+
+class LinearInterpolation:
+    """Creates linear interpolation functions for density and dynamic viscosity
+
+    This class defines two functions that are used to linearly interpolate
+    density (kg/m3) and dynamic viscosity (Pa*S) based on temperature (°C)
+    for the purpose of wind tunnel testing.
+    
+    Attributes:
+        densityFunc
+        viscosityFunc
+
+    Methods:
+        interpolateDensity
+        interpolateViscosity
+    """
+
+    def __init__(self, path):
+        """
+        Reads a CSV file to define two functions for density and dynamic
+        viscosity that can be used with temperature.  The CSV should have same
+        structure as current one in Drive, where the units are in the second row.
+        
+        Parameters:
+            path - path to a CSV file of the air properties
+        """
+        self.logger = logging.getLogger()
```

Comment:
I am hoping this will log with the formatting we have defined already (as the root logger is global within an interpreter).

Reply:
I am not sure about this one either. @Burke-Daniel I would appreciate your opinion on this one.